api.go

```diff
@@ -726,6 +738,17 @@ func (api *API) ImportValue(_ context.Context, req *ImportValueRequest) error {
 	return errors.Wrap(err, "importing")
 }
 
+func importNotNullColumns(index *Index, columnIDs []uint64) error {
+	nnf := index.unprotectedNotNullField()
+	if nnf == nil {
+		return nil
+	}
+
+	notNullRowIDs := make([]uint64, len(columnIDs))
+	notNullTimestamps := make([]*time.Time, len(columnIDs))
```

Comment:
I *think* we can pass in an empty or nil slice here rather than doing this allocation.